.github/workflows/ci.yml

```diff
@@ -55,6 +55,36 @@ jobs:
         env:
           RUST_BACKTRACE: 1
 
+  wasm_build:
+    name: Build wasm32
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Install stable toolchain
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Add wasm target
+        run: rustup target add wasm32-unknown-unknown
+
+       - name: wasm32 build (debug; default features)
+        run: cargo build --target wasm32-unknown-unknown --lib
+        env:
+          RUST_BACKTRACE: 1
```

Comment:
I think the YAML indentation is off here:

```diff
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index d87ccfa..1cbaaff 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -70,10 +70,10 @@ jobs:
       - name: Add wasm target
         run: rustup target add wasm32-unknown-unknown

-       - name: wasm32 build (debug; default features)
+      - name: wasm32 build (debug; default features)
         run: cargo build --target wasm32-unknown-unknown --lib
         env:
-          RUST_BACKTRACE: 1
+           RUST_BACKTRACE: 1

       - name: wasm32 build (debug; all features)
         run: cargo build --target wasm32-unknown-unknown --lib --all-features
```

Reply:
Yeah, I think two-space indentation is expected in YAML?

Reply:
You're both right, sorry about that. It was jut the `- name` line that was mis-indented and it looks good now :+1: